Ogden Lab Virus Recombination Analysis on Click-seq Illumina RNA-seq data
Authored by Alejandra Flores March 2025. Several scripts were generated by Andrew Routh (UTMB) and Alejandra Flores (VUMC).

Data was generated as paired-end fastq files. Trimming from fastq.gz files to separate files by Trimmomatic 0.39. Reference index is built by bowtie-build and ViReMa is run as indicated in Gribble et al, 2020.

Generate junction plots (Junction_Pattern_Plots.R) using BED File (Recombination_Results.bed) as input

Use PRF_calculation.R. Input files are Recombination_results.txt and coverage file.

PCA analysis uses sample_virus_junction.txt as input for each virus

Common junction analysis uses sample_virus_junction.txt as input for each virus

Create Logo plots using Recombination_Results.bed as an input. 

Generate sequence microhomology plots using BED File (Recombination_Results.bed) as input

Positions conserved table (input is Recombination_Results.bed). For the upstream and downstream nucleotide match quantification, empty rows at the end would be counted as a match. Subtract the number of empty rows to the final count.

Frequency of microhomology and usage of microhomology sites for recombination utilized recombination_results.bed as input
